pkg/operator/controller/status/controller.go

```diff
@@ -208,10 +209,10 @@ func (r *reconciler) getOperatorState(nsName string) (operatorState, error) {
 }
 
 // computeOperatorStatusVersions computes the operator's current versions.
-func (r *reconciler) computeOperatorStatusVersions(oldVersions []configv1.OperandVersion, allDNSESAvailable bool) []configv1.OperandVersion {
+func (r *reconciler) computeOperatorStatusVersions(oldVersions []configv1.OperandVersion, dnsProgressing bool) []configv1.OperandVersion {
 	// We need to report old version until the operator fully transitions to the new version.
 	// https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/clusteroperator.md#version-reporting-during-an-upgrade
-	if !allDNSESAvailable {
+	if dnsProgressing {
```

Comment:
Hmm.  If, say, one compute node is down for maintenance, might that block the DaemonSet going Progressing=False?  I'd rather not have the update overall block on that sort of peripheral issue.  On the other hand, we don't want to leave dangling pods with old logic for too long after we've claimed a successful update.  I'm not sure what an elegant situation is for that; might need to happen at the DaemonSet controller level?  Seems like overkill to split into control-plane and compute DaemonSets...